### PR TITLE
Self-managed on-premises only cleanup

### PR DESCRIPTION
**Pull request by bmckercher123**
- Fixes broken links as a result of new section, "Self-managed on-premises"
- Includes fix to move "Enable application layer policy" from Get started to /Security
- Merge to /latest

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 0dc9cfd495c9016d8b13a3122ccf97c93c9359be

https://deploy-preview-3213--calico.netlify.com

File: security/app-layer-policy.md [hunk 1]
```diff
@@ -105,6 +105,6 @@ If the namespace already has pods in it, you must recreate them for this to take
 
 ### Above and beyond
 
-- [Enforce network policy using Istio tutorial]({{ site.baseurl }}/security/tutorials/app-layer-policy/enforce-policy-istio)
-- [Enforce network policy using Istio]({{ site.baseurl }}/security/enforce-policy-istio)
-- [Use HTTP methods and paths in policy rules]({{ site.baseurl }}/security/http-methods)
+- [Enforce network policy using Istio tutorial]({{site.baseurl}}/security/tutorials/app-layer-policy/enforce-policy-istio)
+- [Enforce network policy using Istio]({{site.baseurl}}/security/enforce-policy-istio)
+- [Use HTTP methods and paths in policy rules]({{{site.baseurl}}/security/http-methods)
```
Comment (lmm) [inline security/app-layer-policy.md]: Extra `{`

File: security/enforce-policy-istio.md [hunk 2]
```diff
@@ -25,11 +25,11 @@ This how-to guide uses the following Calico features:
 
 ### Before you begin...
 
-[Install Calico and calicoctl]({{ site.baseurl }}/getting-started/kubernetes/installation/calico)
+[Install Calico and calicoctl]({{ site.baseurl }}/getting-started/kubernetes/)
 
 ### How to
 
-After you complete the steps to [enable application layer policy]({{ site.baseurl }}/getting-started/kubernetes/installation/app-layer-policy), you are ready to create standard Calico network policies with application layer specific attributes.
+After you complete the steps to [Enable application layer policy]({{site.baseurl}}/security/app-layer-policy), you are ready to create standard Calico network policies with application layer specific attributes.
```
[the remark this answers is not in the document]
Comment (lmm) [reply security/enforce-policy-istio.md]: Calico => {{site.prodname}}

File: security/app-layer-policy.md [hunk 3]
```diff
@@ -18,14 +18,14 @@ Enable {{site.prodname}} application layer network policy in Istio service mesh.
...
-For a tutorial on how application layer policy provides second-factor authentication for the mythical Yao Bank, see [Enforce network policy using Istio]({{ site.baseurl }}/security/tutorials/app-layer-policy/enforce-policy-istio).
+For a tutorial on how application layer policy provides second-factor authentication for the mythical Yao Bank, see [Enforce network policy using Istio]({{site.baseurl}}/security/tutorials/app-layer-policy/enforce-policy-istio).
 
 ### Before you begin...
 
 **Required**
 
-- [{{site.prodname}} is installed]({{ site.baseurl }}/getting-started/)
-- [calicoctl is installed and configured]({{ site.baseurl }}/getting-started/calicoctl/install)
+- [Calico is installed]({{site.baseurl}}/getting-started/kubernetes/)
```
[the remark this answers is not in the document]
Comment (bmckercher123) [reply security/app-layer-policy.md]: fixed.

File: security/enforce-policy-istio.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (bmckercher123) [reply security/enforce-policy-istio.md]: fixed.

**Pull request merged**